.github/workflows/publish-docker.yaml

```diff
@@ -17,6 +17,7 @@
 name: publish-docker
 
 on:
+  pull_request:
   push:
     branches:
       - master
```

Comment:
If you want to test this workflow, create a branch in `apache/skywalking-agent-test-tool` repo (not your fork one), and add the branch name to `push.branches`

Reply:
I will submit a new PR